src/utils/mac-table.h

```diff
@@ -53,9 +53,17 @@ struct pg_mac_table {
 };
 
 struct pg_mac_table_iterator {
-	struct pg_mac_table *table;
-	union pg_mac key;
-	int is_end;
+	uint32_t i;
```

Comment:
Could be great to have a small comment about this variable... I'm not sure what's the purpose of i, the name is not that explicit. Same for 5 next lines... Just to say what is gonna be stored inside!